<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sectiona</class>
 <widget class="QWidget" name="sectiona">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>591</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="twSectionA">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>41</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbLenTrainingA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Длительность занятия</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSectionName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelA">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teSectionA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackA">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAllA">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать все</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntConnect">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePlace">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbPlace">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Место проведения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Секции</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
